ansible/playbooks/upgrade.yml

```diff
@@ -305,6 +305,15 @@
         tasks_from: upgrade/main
       when: "'postgresql' in upgrade_components or upgrade_components|length == 0"
 
+- hosts: postgresql
+  become: true
+  become_method: sudo
+  tasks:
+    - import_role:
+        name: upgrade
+        tasks_from: postgres-exporter
+      when: "'postgresql_exporter' in upgrade_components or upgrade_components|length == 0"
```

Comment:
Just thinking if this should be part of the Postgress upgrade path? @to-bar  any thoughts on this?

Reply:
Ok so lets leave it seperate.